src/main/java/cn/nukkit/command/data/BuiltinCommandEnums.java

```diff
@@ -0,0 +1,23 @@
+package cn.nukkit.command.data;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.ToString;
+
+/**
+ * @author lukeeey
+ */
+@Getter
+@ToString
+@RequiredArgsConstructor
+public enum BuiltinCommandEnums {
```

Comment:
Just a quick note: i may remove this in the future in favour of argument classes, e.g. `ItemArgument`